### DISCUSSION TITLE
Statement on Trump Violation of the AGPL and use of FOSS

### DISCUSSION THREAD

**Discussion by wwahammy** [houdiniproject/houdini#826] [General]
Yesterday, it came out that Donald Trump is creating a new social network which appears to be a fork of Mastodon but which is not complying with the terms of the AGPL. Conservancy has [commented on this](https://sfconservancy.org/blog/2021/oct/21/trump-groups-violates-affero-gpl/) and expressed that they are willing to defend users  against Trump's violation.

As the sole AGPL licensed Conservancy project, I believe it would be appropriate to make a statement against Trump's violation of the AGPL. Additionally, as a social good focused FOSS community I believe it would be right to express our opposition to his and his supporters participation in the FOSS community.

Would others in the community support making a statement both opposing his violation of the AGPL and his and his supporters participation in the FOSS community?

**Comment by clarissalimab** [comment]
I support that!

**Reply by tttp** [reply]
Absolutely!

**Comment by wwahammy** [comment]
@kookster any thoughts on this?

**Comment by wwahammy** [comment]
@bkuhn thoughts? Is there anything we need to consider other than making clear that it's a statement from the Houdini community exclusively?
